src/kernels/serviceRegistry.node.ts

```diff
@@ -109,8 +108,6 @@ export function registerTypes(serviceManager: IServiceManager, isDevMode: boolea
     // This will ensure all subsequent telemetry will get the context of whether it is a custom/native/old notebook editor.
     // This is temporary, and once we ship native editor this needs to be removed.
     setSharedProperty('ds_notebookeditor', 'native');
-    const isLocalConnection = serviceManager.get<IServerConnectionType>(IServerConnectionType).isLocalLaunch;
-    setSharedProperty('localOrRemoteConnection', isLocalConnection ? 'local' : 'remote');
```

Comment:
This was not correct anymore, as we can have both local and remote connectsion together